parm/atm/hofx/hofx_nomodel.yaml.j2

```diff
@@ -25,4 +25,8 @@ state:
   state variables: [ua,va,t,delp,sphum,ice_wat,liq_wat,o3mr,phis,
                     slmsk,sheleg,tsea,vtype,stype,vfrac,stc,smc,snwdph,
                     u_srf,v_srf,f10m]
-observations: !INC ${OBS_LIST}
+observations:
+{% filter indent(width=4) %}
```

Comment:
```suggestion
{% filter indent(width=2) %}
```

Reply:
See comment above

Reply:
Indentation for `parm/atm/hofx` templates corrected at b3c8737.

Reply:
I don't think so. 
The obs yamls can be shared (unless the data in them will be different).  The indentation over here in the "calling yam;" is to bring in the "observers" data and place it here relative to `observations`.